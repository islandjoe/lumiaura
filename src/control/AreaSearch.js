import React from 'react'
import From from './area-search/From'
import To from './area-search/To'

import '../layout/AreaSearch.css'

const areaSearch =(props)=> (
  <div className="AreaSearch">
    <From address={ props.home }/>
    <To />
  </div>
)

export default areaSearch