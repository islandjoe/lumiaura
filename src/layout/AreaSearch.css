.AreaSearch {
  width:auto;
}

input[type=text] {
  /* display: block; */
  margin: 6.666px 0;
  width: 100%;
}