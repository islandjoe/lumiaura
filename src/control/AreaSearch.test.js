import React from 'react'
import renderer from 'react-test-renderer'
import {configure, shallow, render, mount} from 'enzyme'
import Adapter from 'enzyme-adapter-react-16'
import AreaSearch from './AreaSearch'

configure({ adapter: new Adapter() })

describe('<AreaSearch/>', ()=> {
  test('should render', ()=> {
    const tree = renderer
      .create(<AreaSearch/>)
      .toJSON()

    expect(tree).toMatchSnapshot()
  })
})