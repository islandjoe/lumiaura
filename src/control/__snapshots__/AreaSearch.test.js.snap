// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AreaSearch/> should render 1`] = `
<div
  className="AreaSearch"
>
  <div
    className="ui input"
  >
    <input
      placeholder="From..."
      type="text"
    />
  </div>
  <div
    className="ui input"
  >
    <input
      placeholder="To..."
      type="text"
    />
  </div>
</div>
`;
